components/demo/demo-page.js

```diff
@@ -3,8 +3,11 @@ import './code-view.js';
 import '../colors/colors.js';
 import '../typography/typography.js';
 import { css, html, LitElement } from 'lit';
+import { getDocumentLocaleSettings, supportedBaseLocales } from '@brightspace-ui/intl/lib/common.js';
```

Comment:
Would you want to use the base locales or the culture-specific ones?

Reply:
I can do either, but I figured the base locales accomplished the task. Is there any advantage to the culture-specific ones?

Reply:
I think either should be fine. I guess the only thing would be if someone wants to check the verbiage for a culture specific locale.

Reply:
Yeah. Ok I'll use the larger set -- need to update my PR in Intl then.

Reply:
Ok -- updated with the culture-specific ones. Screenshots in description updated as well.